cypress/integration/pages/frontPage/tests.js

```diff
@@ -134,6 +134,29 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
             },
           );
         });
+        it('should contain Useful Links if usefulLinks block data exists', () => {
+          cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
+            ({ body }) => {
+              const pageData = body.content.groups;
+              const usefulLinks = pageData.find(data => {
+                return data.type === 'useful-links';
+              });
+              const hasStrapline =
+                usefulLinks &&
+              Object.prototype.hasOwnProperty.call(usefulLinks, 'strapline');
+
+              if (hasStrapline) {
```

Comment:
some linting issues
```
144:20  error  Replace ` hasStrapline ` with `·hasStrapline·`                                                                                                        prettier/prettier
144:20  error  Irregular whitespace not allowed                                                                                                                      no-irregular-whitespace
144:33  error  Irregular whitespace not allowed                                                                                                                      no-irregular-whitespace
145:1   error  Irregular whitespace not allowed                                                                                                                      no-irregular-whitespace
145:1   error  Replace `                usefulLinks ` with `················usefulLinks·`                                                                            prettier/prettier
145:28  error  Irregular whitespace not allowed                                                                                                                      no-irregular-whitespace
146:1   error  Replace `··············Object.prototype.hasOwnProperty.call(usefulLinks, ` with `················Object.prototype.hasOwnProperty.call(usefulLinks,·`  prettier/prettier
146:64  error  Irregular whitespace not allowed                                                                                                                      no-irregular-whitespace
148:1   error  Replace `              if (hasStrapline) ` with `··············if·(hasStrapline)·`                                                                    prettier/prettier
148:1   error  Irregular whitespace not allowed                                                                                                                      no-irregular-whitespace
148:17  error  Irregular whitespace not allowed                                                                                                                      no-irregular-whitespace
148:32  error  Irregular whitespace not allowed                                                                                                                      no-irregular-whitespace
```

Reply:
also instead of using `Object.prototype.hasOwnProperty.call(usefulLinks, 'strapline')`, you can use `'strapline' in usefulLink`, like in the IndexAlso test above on line 99

Reply:
so lines 144 to 146 will turn into `const hasStrapline = usefulLinks && 'strapline' in usefulLinks;` which looks much cleaner